runtime/src/serde_snapshot/tests.rs

```diff
@@ -262,7 +262,7 @@ mod test_bank_serialize {
 
     // These some what long test harness is required to freeze the ABI of
     // Bank's serialization due to versioned nature
-    #[frozen_abi(digest = "ULV2jDndxR3JB677ayyjaamtAcZ24q75tCkHS2bKVoy")]
```

Comment:
Same as earlier, I doubt any of these ABIs changed, check what Trent said, or if that's not what led to these we should identify what did